index.html

```diff
@@ -51,14 +51,37 @@ <h3>Site Preview</h3>
         <!-- Mary is making cards for testimonials here -->
         <section id="testimonials">
             <h3>Testimonials</h3>
+
+        <div class=Container></div>
+
             <div class="cards"></div>
-            <div class="row"></div>
-            <div class="person"></div>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque quibusdam aliquam inventore fuga porro
-                dicta perspiciatis quia nihil tempora quos odio corrupti, eius, distinctio error quaerat itaque
-                voluptatem atque reiciendis. Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque quibusdam
-                aliquam inventore fuga porro dicta perspiciatis quia nihil tempora quos odio corrupti, eius, distinctio
-                error quaerat itaque voluptatem atque reiciendis.</p>
+                <div class="nameHere"></div>
+                    <div class="person"></div>
+                        <div img src= https://mail.google.com/mail/u/0?ui=2&ik=2f75f0ec65&attid=0.4&permmsgid=msg-a:r-2937301285572270881&th=18751ab278f62fd1&view=fimg&fur=ip&sz=s0-l75-ft&attbid=ANGjdJ-l-Q_kgBDFL3ZtvsvaMgo6JkLABvpo8BqACewsfW2CAVA--murunUDknnuwq4OKtqSwKOysPU2Y1katoS5s6nfg2OEmhERGi2Sbjtg7MfrOdH5yHfFBHOchUs&disp=emb&realattid=ii_lg3qrw4h5></div
```

Comment:
These images shouldn't be nested inside of the opening tag of the div - the img element is self-closing so should look like this: 

<img src="url-goes-here">

Reply:
I got a 404 to the link sent

Reply:
thanks